resources/app/log-collector.yaml

```diff
@@ -103,7 +103,7 @@ spec:
     spec:
       priorityClassName: system-cluster-critical
       securityContext:
-        runAsUser: 0
+        runAsUser: -1
```

Comment:
I thought `-1` was a rigging or gravity think dima added, not sure it works with normal kubectl?

Reply:
I believe gravity tool is doing that job. In this case, it happens in `gravity app unpack` in initContainer for job hook.

Reply:
Only resources stored as YAML files are subject to automatic translation.  If
a Cluster life cycle hook uses custom resource provisioning, it might need to perform
the conversion manually.

The value of the effective service user ID is stored in the
`GRAVITY_SERVICE_USER` environment variable which is available to each
hook.


From the docs, and source code for sure proves it.